core/src/test/scala/unit/kafka/server/DynamicBrokerConfigTest.scala

```diff
@@ -713,6 +714,99 @@ class DynamicBrokerConfigTest {
     config.updateCurrentConfig(new KafkaConfig(props))
     assertFalse(config.nonInternalValues.containsKey(KafkaConfig.MetadataLogSegmentMinBytesProp))
   }
+
+  @Test
+  def testDynamicLogLocalRetentionMsConfig(): Unit = {
+    val props = TestUtils.createBrokerConfig(0, TestUtils.MockZkConnect, port = 8181)
+    props.put(KafkaConfig.LogRetentionTimeMillisProp, "1000")
```

Comment:
As these values are long types. Can we have a value more than Integer.MAX? Same comment in this class for other retention properties.